### DISCUSSION TITLE
Keeping kernels running after disconnections

### DISCUSSION THREAD

**Discussion by david-cortes** [microsoft/vscode-jupyter#11103] [Questions and Answers] [answered]
When using the extension in [code-server](https://github.com/coder/code-server) (a fork of vscode that allows accessing it remotely in a web browser), when I hit "refresh" or when the internet connection dies and then returns, running jupyter kernels are gone.

Is there some way of preserving running kernels across refreshes and connection interruptions?

**Comment by DonJayamanne** [comment, marked as answer]
Currently there isn't a way, this feature is planned here https://github.com/microsoft/vscode-jupyter/issues/3998
Please upvote that issue
